### DISCUSSION TITLE
Is there a way to set all nodes to terrain height?

### DISCUSSION THREAD

**Discussion by evenmorestupiderhats** [KillBait/AutoDrive_Course_Editor#32] [Q&A]
After copying and pasting some nodes, some nodes are above the terrain and others are below the terrain.  Is there a feature in Course Editor to set node height to the terrain height?

**Comment by KillBait** [comment]
You can do it a few  ways.

1. If you want all the nodes at the same height you can use the depth align
option

2. If the nodes have a green triangle above them you can use the fix height
menu option ( if the height map for the config is loaded)

3. You can try putting the nodes above the vehicle height in game and you
may be able to move them and they should align to the terrain

4. Try enabling debug mode from the help menu and select 'show node
location' and when you hover over the nodes look at the Y value,  you can
manually edit the location and adjust them to the correct location, but you
will have to do each node in turn

Hope this helps

**Reply by evenmorestupiderhats** [reply]
Option 2 doesn't seem to be working even if I set a node's Y value to 0.0.  The program seems to show the heightmap correctly, but there aren't any nodes that have a green triangle.  
![image](https://user-images.githubusercontent.com/64505157/149861818-7690f8ee-4f76-4a37-b26e-2ceafd75f09c.png)

**Comment by evenmorestupiderhats** [comment]
"if the height map for the config is loaded"

  I think this is what I'm missing.   Thank you! I'll try this.

**Comment by KillBait** [comment]
Try setting 1 node to the correct Y value and use the depth align on all the others, that should temporary fix them.

I will take a look at the copy paste code when I have time today,, im at work currently so I can't check right now
